clients/da-vinci-client/src/test/java/com/linkedin/davinci/store/rocksdb/RocksDBSstFileWriterTest.java

```diff
@@ -289,108 +268,6 @@ public void testOpenAndSyncWithAllValidFiles() throws IOException {
     }
   }
 
-  @Test
-  public void testCreateSnapshotWithBlobTransferDisabled() throws RocksDBException {
```

Comment:
Since we are moving codes hence deleting these codes from old classes, can we also make sure they are added to the new classes?

Reply:
Move the unit test to RocksDBStoragePartitionTest to test the blob transfer config.

Reply:
Hmm. I think it makes more sense to have them in `BlobSnapshotManagerTest` since the method `createSnapshotForBatch` is in the `BlobSnapshotManager`. Having them in `RocksDBStoragePartitionTest` also works but that looks like a bit convoluted given we have to prepare many things for testing. What do you think?

Reply:
added in both BlobSnapshotManagerTest/RocksDBStoragePartitionTest